### ISSUE TITLE
libusb WARNING logged when using J-Link

### ISSUE THREAD

**Issue by ksigurdsson** [pyocd/pyOCD#994]
I'm connecting to my target via a SEGGER J-Link.
I get the following warning about STLink and CMSIS-DAPv2 probes not being supported.
[ WARNING] STLink and CMSIS-DAPv2 probes are not supported because no libusb library was found

Is there a way to prevent this warning when using a J-Link?

Thanks

**Comment by flit** [comment]
I agree the warning isn't ideal. The problem is that at the point where the warning is issues, pyOCD doesn't know that you are using a J-Link. The warning is intended to help the majority of users using CMSIS-DAP/STLink probes in the case where they didn't install libusb—even though their board is connected it doesn't show up in pyOCD.

One solution I've been thinking about is to create a Python package that contains the libusb libraries for the major OSes. PyOCD could depend on this rather than requiring users to install libusb. And thus the libusb warning would no longer be required.
